archetypes/archetype-lambda/src/main/resources/global.vm

```diff
@@ -0,0 +1,26 @@
+## global variables used by the project
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+## TODO: find a better way to handle this
+#if( $service == 'dynamodb')
+    #set ( $service = 'dynamoDb')
+#end
```

Comment:
Ouch. Is this the only one?

Reply:
No, this would not be the only one. I'll think about a better way to map serviceId -> serviceClientName

Reply:
Can't think of an easy way to do it. Currently, the service module name is lower case derived from serviceId, and the client class name is camel case derived from service name.

We can probably write some scripts to search service-2.json files to map serviceId to serviceName, but that seems a bit overkill. I'm leaning towards just fixing the popular services here.